src/store/AuthContext.jsx

```diff
@@ -29,16 +29,53 @@ export const AuthContextProvider = (props) => {
     }
    };
 
-   const login = () => {
-        setUser({
-            userName: "Monique",
+   const login = (email, password) => {
+        console.log("email, password", email, password);
+
+        signInWithEmailAndPassword(auth, email, password)
+        .then((userCredential) => {
+            const loggedUser = userCredential.user;
+            console.log("user", loggedUser);
+            setUser(loggedUser);
         })
-   }
+        .catch ((error) => {
+            const errorCode = error.code;
+            const errorMessage = error.message;
+            console.log(`Error: ${errorCode} - ${errorMessage}`);
+        });
+        }; 
+
+        const checkUserStatus = () => {
+            onAuthStateChanged(auth, (user) => {
+                if (user) {
+                  
+                  //const uid = user.uid;
```

Comment:
Are you using this status?
The `user.uid` os a good property to use in order to validate requests